docs/_config.yml

```diff
@@ -4,10 +4,10 @@ output: web
 # this property is useful for conditional filtering of content that is separate from the PDF.
 # note that this adaption of Tom Johnston's Jekyll documentation theme does not generate PDF outputs
 
-topnav_title: Zowe CLI CICS deploy plug-in
+topnav_title: Zowe CLI CICS® deploy plug-in
```

Comment:
Is trademarking page titles necessary? I don't see this being done in the CICS KC.